### PR TITLE
added ticker component

### PR DESCRIPTION
**Pull request by saxenapeeyush**

### CODE REVIEW

File: src/components/atoms/Ticker/Tick/index.tsx [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+    currentClassName,
+    hiddenClassName
+  } = props;
+
+  const index = range.indexOf(value);
+
+  return (
+    <span className={`ticker42Tick ${className}`}>
+      <span
+        className={`ticker42hiddenPiece ${className}`}
```
Comment (ankitchouhan1020) [inline src/components/atoms/Ticker/Tick/index.tsx:18]: same className ?

File: src/components/atoms/Ticker/ticker.css [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+.ticker42Tick {
+  overflow: hidden;
+  display: inline-block;
+  position: relative;
+}
+
+.ticker42Piece {
+  position: absolute;
+  left: 0;
+  z-index: 10;
```
Comment (ankitchouhan1020) [inline src/components/atoms/Ticker/ticker.css:23]: we can discuss this.

File: src/components/atoms/Ticker/Tick/index.tsx (hunk 1 above)
Comment (saxenapeeyush) [reply src/components/atoms/Ticker/Tick/index.tsx]: discussed already and changed in the file.

File: src/components/atoms/Ticker/ticker.css (hunk 2 above)
Comment (saxenapeeyush) [reply src/components/atoms/Ticker/ticker.css]: discussed already and changed in the file.

**Review by ankitchouhan1020 — APPROVED** [review]
We will update this component in future with changes in the original 3rd party lib.

**Review by ghost — APPROVED** [review]
This is a third party code only right?

**Comment by saxenapeeyush** [comment]
@vyapak1209 yes it is third party code only with some refactoring.

**Review by dheerajsingh123 — APPROVED** [review]

**Pull request merged**